doc/config.md

```diff
@@ -94,7 +93,8 @@ config file at runtime.
     "ResendDirectAnnounce": true,
     "StoreBatchSize": 4096,
     "SyncSegmentDepthLimit": 2000,
-    "SyncTimeout": "2h0m0s"
+    "SyncTimeout": "2h0m0s",
+    "SyncWriteEntries": false
```

Comment:
Does this mean entries never get synced?

Reply:
No, it means that we are not forcing it to write synchronously, and allowing the default async behavior.  Better name suggestion? (I would like a default value of `false`)

Reply:
OK we have the word "Sync" in `SyncTimeout` implying the meaning "fetch", and right below it `SyncWriteEntries` implying synchronous.

Would it make sense to rename either so that it is less confusing? Or consistently use one or the other to mean the same thing.